donut/modules/arcfeedback/templates/complaint.html

```diff
@@ -0,0 +1,168 @@
+			    <div class="form-group">
+				<p>Remove Email (or comma separated list of emails): </p>
+				<input class="form-control" type="text" name="email" />
+			    </div>
+			    <input class="btn btn-primary" type="submit" value="Remove" id="remove-email-btn">
+			</form>
+		    </div>
+		</div>
+	    </div>
+	    {% endif %}
+	    <div class="row">
+		<div class="col-md-8 col-md-offset-2">
+		    <div>
+			<table class="table table-bordered" id="messageList">
+			    <thead class="thead-default">
+				<tr> <th>Time Posted</th> <th>Name</th> <th>Message</th> </tr>
+			    </thead>
+			    {% for message in complaint['messages'] %}
+			    <tr> 
+				<th>{{message['time'].strftime('%b %d %Y %-I:%M%p')}}</th> 
+				<th>{{ message['poster'] }}</th> 
+				<td>{{ message['message'] }}</td>
+			    </tr>
+			    {% endfor %}
+			</table>
+		    </div>
+		    <div class="panel panel-default">
+			<div class="panel-heading">
+			    <p>Add a comment<p>
+			</div>
+			<div class="panel-body">
+			    <form method="POST" id="addMessage" action="/">
+				<div class="form-group">
+				    <p>Name: </p>
+				    <input class="form-control" type="text" name="poster" maxlength="50" />
+				</div>
+				<div class="form-group">
+				    <p>Message: </p>
+				    <textarea class="form-control" name="message" rows="10" required maxlength="5000"></textarea>
+				</div>
+				<input class="btn btn-primary" id="submitBtn" type="submit" value="Submit" />
+			    </form>
+			</div>
+		    </div>
+		</div>
+	    </div>
+	</div>
+    </div>
+</div>
+{% endblock %}
+{% block scripts %}
+{{ super() }}
+<script>
+ function addMessage(response) {
+     var msg = response['message'];
+     var poster = response['poster'];
+     var time = response['time'];
+     var msg_html = "<tr> <td>" + time + "</td> <td>" + poster + "</td> <td>" + msg + "</td> </tr>";
+     $('#messageList').append(msg_html);
+ }
+ 
+ function addMessageError(response) {
+     $(messageList).append(response);
+ }
+
+ $(document).ready(function() {
+     {% if 'TOGGLE_READ' in perms %}
+         {% if complaint['status'] == 'new_msg' %}
+         $('#markUnread').hide();
+         {% else %}
+         $('#markRead').hide();
+         {% endif %}
+     {% else %}
+     $('#markUnread').hide();
+     $('#markRead').hide();
+     {% endif %}
+     $('#addMessage').on('submit', function(e) {
+	 e.preventDefault();
+	 $.ajax({
+	     url: "{{ url_for('arcfeedback.arcfeedback_add_msg', id=complaint['uuid']) }}",
+	     data: $('#addMessage').serialize(),
+	     type: 'POST',
+	     success: addMessage,
+	     error: addMessageError
+	 }); 
+	 $('#addMessage')[0].reset()
+     });
+     $('#markRead').on('click', function() {
+	 $.ajax({
+	     url: "{{ url_for('arcfeedback.arcfeedback_mark_read', id=complaint['uuid']) }}",
+	     type: "GET",
+	     success: function(){
+		 $('#markRead').hide();
+		 $('#markUnread').show();
```

Comment:
page doesn't refresh to show updated status

Reply:
No -- I'm using ajax here to send the status update to the server and javascript to update the page without refreshing